### PR TITLE
ENG-2545 Change fragment code validation to accept the dash character

### PR DESCRIPTION
**Pull request by raxkaynan**

### CODE REVIEW

File: src/ui/fragments/common/FragmentForm.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ import {
 
 const maxLength50 = maxLength(50);
 
+const fragmentCode = value => (
+  value && /^[0-9a-zA-Z_\-.]+$/i.test(value) ?
```
Comment (gReis89) [inline src/ui/fragments/common/FragmentForm.js:22]: if `value` is `0`, wouldn't it brake this logic, since `0` is a falsy value?

Comment (raxkaynan) [reply src/ui/fragments/common/FragmentForm.js:22]: Had to double-check it to be sure 😄. But no it doesn't since `value` is a string, not a number.

**Review by gReis89 — APPROVED** [review]

**Comment by user-54dbb57a** [comment]
Rebased to the latest develop to get the latest pipelines updates

**Comment by raxkaynan** [comment]
Closed since ticket has been rejected.

**Pull request closed without merging**